### PR TITLE
disable biometric auth when device can't save password (rooted)

### PR DESCRIPTION
**Pull request by yenda**
ready

### CODE REVIEW

File: src/status_im/multiaccounts/biometric/core.cljs [hunk 1]
```diff
@@ -101,7 +101,13 @@
 (re-frame/reg-fx
  :get-supported-biometric-auth
  (fn []
-   (get-supported #(re-frame/dispatch [:init.callback/get-supported-biometric-auth-success %]))))
+   (let [callback #(re-frame/dispatch [:init.callback/get-supported-biometric-auth-success %])]
+     ;;NOTE: if we can't save user password, we can't support biometrics
+     (keychain/can-save-user-password?
+      (fn [can-save?]
+        (if can-save?
+          (get-supported callback)
```
Comment (cammellos) [inline src/status_im/multiaccounts/biometric/core.cljs:109]: where is `get-supported` defined, do you mean `callback` here?

Comment (cammellos) [reply src/status_im/multiaccounts/biometric/core.cljs:109]: nevermind, I thought it was a `let` block, blind as a bat :)

**Review by cammellos — APPROVED** [review]

**Review by rasom — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by churik** [comment]
@yenda 
I didn't find other issues, but I suggest to wait fix for https://github.com/status-im/status-react/issues/10061 (or fix it in this PR) - due to it I can't test part of this PR as well

**Comment by churik** [comment]
this PR fixes #5674

**Comment by churik** [comment]
@yenda can you please rebase it to current develop?

**Comment by churik** [comment]
@yenda can we finish this one?:)
I need rebased builds and take them again into testing

**Comment by yenda** [comment]
@churik done

**Comment by churik** [comment]
Tested with phone with Touch ID, with phone when all biometric data is deleted and on emulator.
No issues are found.

**Pull request merged**